backend/node_engine/workflow_engine.py

```
@@ -128,8 +128,8 @@ async def execute_workflow(
                 # Validate inputs
                 validation_result = node.validate_inputs()
                 if not validation_result["valid"]:
-                    error_msg = f"Node {node_id} input validation failed: {
-                        ', '.join(validation_result['errors'])}"
+                    error_msg = f"Node {node_id} input validation failed: "
+                    f"{', '.join(validation_result['errors'])}"
                     context.add_error(node_id, error_msg)
                     raise WorkflowExecutionException(error_msg)
```

Ruff (0.14.1)

Abstract `raise` to an inner function

(TRY301)